<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinancialManagementScreen</class>
 <widget class="QDialog" name="FinancialManagementScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tela de Gerenciamento Financeiro</string>
  </property>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Giro de Gado</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>460</y>
     <width>531</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reportButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>531</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Relatório</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>531</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Exclusão</string>
   </property>
  </widget>
  <widget class="QPushButton" name="queryButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>531</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Consulta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>531</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cadastro</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
